hw_4/Task3.js

```diff
@@ -0,0 +1,22 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+
+
+const minAge = 18;
+const maxAge = 60;
+const age = prompt("What's your age?");
+
+if (isNaN(Number(age))){
```

Comment:
convert age to number not under if, but before/after to use number value in checks
```suggestion
const age = +prompt("What's your age?");
if (isNaN(age)){
```